### ISSUE TITLE
Making this site into a PWA

### ISSUE THREAD

**Issue by tmsagarofficial** [pushkaraj2007/Threadify#47]
I want to let u know that this project will, be great as a PWA and hence I can work on doing this. Looking forward for more contributions in this project.

**Comment by tmsagarofficial** [comment]
Check and let me know if i can start working on this @pushkaraj2007

**Comment by pushkaraj2007** [comment]
@tmsagarofficial 

You know about the recent changes in the price of Twitter API, right?

So we will not be able to use Twitter API now. So we have to make this app such that there will be no involvement of Twitter API.

We can make the user to paste the first Tweet and the link of the Tweet.

By this we can only continue this app, otherwise there is no other option. 

We can discuss more about how this project can be continued.

BTW, you can start working on this feature.

**Comment by pushkaraj2007** [comment]
You can also contact me on Twitter to discuss further about development of this project.

Twitter profile: twitter.com/pushkaraj2007

**Comment by tmsagarofficial** [comment]
Yeah will contact you

> You can also contact me on Twitter to discuss further about development of this project.
> 
> Twitter profile: twitter.com/pushkaraj2007

**Comment by tmsagarofficial** [comment]
> @tmsagarofficial
> 
> You know about the recent changes in the price of Twitter API, right?
> 
> So we will not be able to use Twitter API now. So we have to make this app such that there will be no involvement of Twitter API.
> 
> We can make the user to paste the first Tweet and the link of the Tweet.
> 
> By this we can only continue this app, otherwise there is no other option.
> 
> We can discuss more about how this project can be continued.
> 
> BTW, you can start working on this feature.

Yeah fortunately or unfortunately we need to incorporate this change in the flow of approach to keep this amazing project alive.

**Comment by pushkaraj2007** [comment]
Yes :(

**Comment by pushkaraj2007** [comment]
Do you want to work on this @tmsagarofficial.

If anyone else want to work on this issue it's also fine fine if Sagar don't.

**Comment by tmsagarofficial** [comment]
Ill work on it @pushkaraj2007

**Comment by pushkaraj2007** [comment]
Go ahead @tmsagarofficial

**Comment by tmsagarofficial** [comment]
> @tmsagarofficial
> 
> You know about the recent changes in the price of Twitter API, right?
> 
> So we will not be able to use Twitter API now. So we have to make this app such that there will be no involvement of Twitter API.
> 
> We can make the user to paste the first Tweet and the link of the Tweet.
> 
> By this we can only continue this app, otherwise there is no other option.
> 
> We can discuss more about how this project can be continued.
> 
> BTW, you can start working on this feature.

Lets close this issue as of now then ..
I sorry to see this happen @pushkaraj2007
